mods/ra/weapons/missiles.yaml

```diff
@@ -448,6 +308,7 @@ SubMissileAA:
 		ValidTargets: Air, Ground, Water
 
 SCUD:
+	Inherits: ^AntiGroundMissile
```

Comment:
It might be worth adding a `-Projectile:` before the bullet definition to wipe out the old Missile keys.